site/en/tutorials/images/data_augmentation.ipynb

```diff
@@ -0,0 +1,599 @@
+        "## Overview\n",
+        "\n",
+        "When training a Machine Learning model, especially Deep Learning models, We require a lot of data. If there is a scarcity of data, then the model is prone to underfitting. Unfortunately, scarcity of data is very common. So, we have to make use of the handful amount of data we have. Also, creating a variation in data helps in preventing overfitting as well.\n",
+        "\n",
+        "In this scenario, using data augmentation, We could increase the number of examples of a dataset. We take the provided data and augment it i.e. crop the image, rotate it, change colors, flip, etc. Doing this increases the number of samples and increase variation and helps us to create a good model."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8sZIVqk7HvnC",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Setup"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "r-vdtEaCeK4Z",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3TS4vBJBd8jY",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!pip install -q tensorflow_addons"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "rdP8EQbPsyRA",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "try:\n",
+        "  %tensorflow_version 2.x\n",
+        "except:\n",
+        "  pass\n",
+        "\n",
+        "import urllib\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "from tensorflow.keras.datasets import mnist\n",
+        "from tensorflow.keras import layers\n",
+        "\n",
+        "import tensorflow_addons as tfa\n",
+        "import tensorflow_datasets as tfds\n",
+        "\n",
+        "import matplotlib.pyplot as plt\n",
+        "import matplotlib as mpl\n",
+        "mpl.rcParams['figure.figsize'] = (12, 5)\n",
+        "\n",
+        "import numpy as np"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "__EuXwM-8uth",
+        "colab_type": "text"
+      },
+      "source": [
+        "We will be checking the data augmentation feautres on an image and we will be augmenting a whole dataset later to train a model."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "frBSdODBLOOI",
+        "colab_type": "text"
+      },
+      "source": [
+        "Download [this image](https://commons.wikimedia.org/wiki/File:Felis_catus-cat_on_snow.jpg), by Von.grzanka, for augmentation."
```

Comment:
Can you display the image inline? Most readers won't click through. Or maybe present this closer to where you display it below